Scoop/NetworkManager.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  NetworkManager.swift
+//  Scoop
+//
+//  Created by Richie Sun on 2/8/23.
+//
+
+import Foundation
+import Alamofire
+
+struct NetworkManager {
```

Comment:
Just curious why are you making NetworkManager a struct instead of a class? Would that still work with the singleton pattern?